README.md

```diff
@@ -35,6 +35,15 @@ To run the `reporter`:
 brownie run reporter report
 ```
 
+Reports generated explained for Dummies
+**address.csv**: All voting addresses
```

Comment:
can't see where this comes from :thinking:

Reply:
Sorry, I don't get the question. If I have to guess I understand your concern is where do these csv come from? those are the generated on a monthly basis, the ones you share on discord. Specifically addresses.csv shows all addresses with voting power. Hope that somehow clarifies it